pom.xml

```diff
@@ -144,6 +144,57 @@
+                        <configuration>
+                            <title>Kafka Connect HDFS</title>
+                            <logo>logos/hdfs.jpeg</logo>
+                            <sourceUrl>https://github.com/confluentinc/kafka-connect-hdfs</sourceUrl>
+                            <documentationUrl>https://docs.confluent.io/current/connect/connect-hdfs/docs/index.html</documentationUrl>
+                            <description>
+                                The HDFS connector allows you to export data from Kafka topics to HDFS files in a variety of formats and integrates with Hive to make data immediately available for querying with HiveQL.
+
+                                The connector periodically polls data from Kafka and writes them to HDFS. The data from each Kafka topic is partitioned by the provided partitioner and divided into chunks. Each chunk of data is represented as an HDFS file with topic, kafka partition, start and end offsets of this data chunk in the filename. If no partitioner is specified in the configuration, the default partitioner which preserves the Kafka partitioning is used. The size of each data chunk is determined by the number of records written to HDFS, the time written to HDFS and schema compatibility.
+
+                                The HDFS connector integrates with Hive and when it is enabled, the connector automatically creates an external Hive partitioned table for each Kafka topic and updates the table according to the available data in HDFS.
+                            </description>
+
+                            <supportProviderName>Confluent, Inc.</supportProviderName>
+                            <supportSummary>Confluent supports the HDFS sink connector alongside community members as part of its Confluent Platform open source offering.</supportSummary>
+                            <supportUrl>https://docs.confluent.io/current/</supportUrl>
+                            <supportLogo>logos/confluent.png</supportLogo>
+
+                            <ownerUsername>confluentinc</ownerUsername>
+                            <ownerType>organization</ownerType>
+                            <ownerName>Confluent, Inc.</ownerName>
+                            <ownerUrl>https://confluent.io/</ownerUrl>
+                            <ownerLogo>logos/confluent.png</ownerLogo>
+
+                            <pluginTypes>
+                                <pluginType>sink</pluginType>
+                            </pluginTypes>
+
+                            <tags>
+                                <tag>hadoop</tag>
+                                <tag>hdfs</tag>
+                                <tag>hive</tag>
+                            </tags>
+
+                            <requirements>
+                                <requirement>Confluent Platform installed and running</requirement>
+                                <requirement>Hadoop installed and running</requirement>
```

Comment:
Probably sufficient to just specify the Hadoop versions rather than say "... installed and running".

Reply:
Couldn't find anything hard on what versions of Hadoop the connector works with (I know we have `2.7.3` specified in the POM but not sure about the implications for forwards/backwards compatibility with other versions). Unless we can provide a useful version range here it might make more sense to just eliminate the requirement altogether, since just specifying `Hadoop` seems obvious.